docs/initial-setup-f3e7fa9.md

```diff
@@ -21,41 +21,78 @@ Create an instance of the Application Autoscaler service and bind it to your app
 
 
 ## Procedure
-
+### Bind your application
 1.  Check if the Application Autoscaler service is listed in the Service Marketplace using the following command:
 
-    ```
+    ```shell
     cf marketplace
     ```
 
 2.  Create an instance of the service using the following command:
 
-    ```
+    ```shell
     cf create-service autoscaler <service plan name> <instance name>
     ```
 
-    > ### Sample Code:  
-    > ```
+    > ### Sample Code:
+    > ```shell
     > cf create-service autoscaler standard myservice
     > ```
 
 3.  Bind the service instance to your application using the following command. You can bind as many applications as you want to an Application Autoscaler service instance.
 
-    ```
+    ```shell
     cf bind-service <application name> <instance name> -c <file name>.json
     ```
 
     The JSON file contains the policy, which is needed to initiate the scaling of an application. For more information about how to define a policy, see [Defining a Scaling Policy](defining-a-scaling-policy-79f443a.md).
 
-    > ### Note:  
+    > ### Note:
     > If you are building a multitarget application \(MTA\), you can leverage the MTA deployment descriptor to provide the scaling policy. See [Service Binding Parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/c7b09b79d3bb4d348a720ba27fe9a2d5.html).
 
 4.  Check if the instance is successfully bound to the application using the following command:
 
-    ```
+    ```shell
     cf service <instance name>
     ```
 
     The *Bound apps* field displays all applications bound to the instance.
 
-
+### Install Autoscaler's plug-in for the “Cloud Foundry CLI”
+The plug-in can easily be installed directly via the command-line:
+```shell
+cf install-plugin -r CF-Community app-autoscaler-plugin
+```
+
+For further installation-options, please look into the [documentation](<https://github.com/cloudfoundry/app-autoscaler-cli-plugin#install-plugin>) on its official repository on GitHub.
+
+### Attach a policy
+ 1. Create a file named “app-policy.json” with the following content:
+    ```json
+    {
+      "instance_min_count": 1,
+      "instance_max_count": 2,
+      "scaling_rules": [
+        {
+          "metric_type": "cpuutil",
+          "breach_duration_secs": 120,
+          "threshold": 70,
+          "operator": ">=",
+          "cool_down_secs": 120,
+          "adjustment": "+1"
+        },
+        {
+          "metric_type": "cpuutil",
+          "breach_duration_secs": 120,
+          "threshold": 25,
+          "operator": "<",
+          "cool_down_secs": 120,
+          "adjustment": "-1"
+        }
+      ]
+    }
+    ```
+ 2. Attach the policy:
+    ```shell
+    cf attach-autoscaling-policy <application name> 'app-policy.json'
+    ```
```

Comment:
As we already have attached a policy through binding with it in step 3, we can remove this part.
```suggestion
```